### ISSUE TITLE
Console showing letters as you type, but one behind.

### ISSUE THREAD

**Issue by georgiamshaw** [fac-17/Puteam-Week4#36]
We can see the letters appearing in the console as you type them! But they're one behind. Not sure how to fix this but Colette managed to do it for us so ask her :) Jack suggested using keyup instead of keydown. 
<img width="1440" alt="Screenshot 2019-07-26 at 10 46 58" src="https://user-images.githubusercontent.com/36238916/61943178-d5e2b000-af92-11e9-8141-cffbb565df06.png">

**Comment by Coletterbox** [comment]
Yep, the fix was keyup instead of keydown (since keydown triggered the part that saved the contents of the input box as a variable before the new letter was added). Might be the same for you guys - I don't remember your code.
